avocado/plugins/spawners/podman.py

```diff
@@ -176,6 +184,16 @@ async def spawn_task(self, runtime_task):
 
         return returncode == 0
 
+    async def create_task_output_dir(self, runtime_task):
+        task = runtime_task.task
+        output_dir_path = os.path.join(self.job_output_dir,
+                                       task.identifier.str_filesystem)
+        output_podman_path = os.path.join('/tmp', 'avocado-results')
+
+        os.makedirs(output_dir_path, exist_ok=True)
+        runtime_task.task.setup_output_dir(output_podman_path)
+        await self._create_container_for_task(runtime_task, output_dir_path)
```

Comment:
While is ok to call other methods inside a bigger method like "spawn_task", I find odd having this inside this method.

Based on this method name, I was not expecting to create dir *plus* creating a container as well. Maybe this call should be outside of this method. This could return the output_dir_path, so that the upper caller could be responsible to passing this to the _create_cotainer_for_task. What do you think?

Reply:
sure I will change that.